modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -60,6 +53,33 @@ class PandasOnRayDataframeVirtualPartition(PandasDataframeAxisPartition):
     instance_type = ray.ObjectRef
     axis = None
 
+    # these variables are intentionally initialized at runtime (see #6023)
+    _DEPLOY_AXIS_FUNC = None
+    _DEPLOY_SPLIT_FUNC = None
+    _DRAIN_FUNC = None
+
+    @classmethod
+    def _get_deploy_axis_func(cls):  # noqa: GL08
+        if cls._DEPLOY_AXIS_FUNC is None:
+            cls._DEPLOY_AXIS_FUNC = RayWrapper.put(
+                PandasDataframeAxisPartition.deploy_axis_func
+            )
+        return cls._DEPLOY_AXIS_FUNC
```

Comment:
maybe use a [cached property](https://docs.python.org/3/library/functools.html#functools.cached_property):
```python
# somewhere at the top
import functools
# skip
class Deployers:
    @functools.cached_property
    def axis_func(self):
        return RayWrapper.put(PandasDataframeAxisPartition.deploy_axis_func)
    # etc.
_DEPLOYERS = Deployers()
# now use it
def foo():
    _DEPLOYERS.axis_func(args)
```

Reply:
Doesn't work, some problems with serialization: https://github.com/modin-project/modin/actions/runs/4765017932/jobs/8470382071

Reply:
Do note that I have introduced a _separate_ class which I've **instantiated** and used values from its _instance_, not put these as properties into `PandasOnRayDataframeVirtualPartition` itself. It should have worked that way 😉